packages/sit-onyx/src/components/OnyxNavButton/OnyxNavButton.stories.ts

```diff
@@ -0,0 +1,84 @@
+import { defineStorybookActionsAndVModels } from "@sit-onyx/storybook-utils";
+import type { Meta, StoryObj } from "@storybook/vue3";
+import { h } from "vue";
+import OnyxNavButton from "./OnyxNavButton.vue";
+import OnyxListItem from "../OnyxListItem/OnyxListItem.vue";
+import OnyxBadge from "../OnyxBadge/OnyxBadge.vue";
+
+/**
+ * The nav button is used internally to build the main navigation bar component and is not intended to be used individually.
+ */
+const meta: Meta<typeof OnyxNavButton> = {
+  title: "support/NavButton",
+  ...defineStorybookActionsAndVModels({
+    component: OnyxNavButton,
+    events: ["click"],
+    argTypes: {
+      default: {
+        control: { type: "text" },
+      },
+    },
+  }),
+};
+
+const nestedChildren = [
+  { label: "Nested Item 1", href: "#", active: true },
+  { label: "Nested Item 2", href: "https://onyx.schwarz" },
+  { label: "Nested Item 3", href: "#3" },
+  { label: "Nested Item 4", href: "#4" },
+  { label: "Nested Item 5", href: "#5" },
+];
+
+export default meta;
+type Story = StoryObj<typeof OnyxNavButton>;
+
+/**
+ * This example shows a default nav button.
+ */
+export const Default = {
+  args: {
+    label: "Nav Button",
+    default: "Item",
+  },
+} satisfies Story;
+
+/**
+ * This example shows an active nav button.
+ */
+export const Active = {
+  args: {
+    ...Default.args,
+    active: true,
+  },
+} satisfies Story;
+
+/**
+ * This example shows the nav button with nested children.
+ */
+export const WithChildren = {
```

Comment:
The example is cut off when opening the flyout. We propably need to add a decorator here to set height/padding

![image](https://github.com/SchwarzIT/onyx/assets/67898185/c45b8e18-7cd1-4514-9dd5-3ddfb0308ea1)